openbas-model/src/main/java/io/openbas/database/model/Team.java

```diff
@@ -123,9 +135,11 @@ public class Team implements Base {
       orphanRemoval = true)
   @JsonProperty("team_exercises_users")
   @JsonSerialize(using = MultiModelDeserializer.class)
+  @Schema(description = "List of 3-tuple linking simulations and users to this team")
```

Comment:
With a multiIdListDeserializer, this is only the list of ID's on client side, no ?

Reply:
Yes, I added this in the description